web-app/public/locales/fr/common.json

```diff
@@ -2,8 +2,8 @@
     "copyToClipboard": "Copier dans le presse-papiers",
     "copied": "Copié!",
     "name": "Nom",
-    "email": "E-mail",
+    "email": "Courriel",
```

Comment:
Is this a valid translation, or will e-mail work?

Reply:
Yeah, courriel is the word we use in French.